brands/shop/bicycle.json

```
@@ -42,6 +42,15 @@
       "shop": "bicycle"
     }
   },
+  "shop/bicycle|Stella": {
+    "locationSet": {"include": ["be", "de", "nl"]},
+    "tags": {
+      "brand": "Stella",
+      "brand:wikidata": "Q98299138",
+      "name": "Stella E-Bikes",
```

Are you ok if we just set the name to "Stella"?   I searched for some imagery of storefronts and I don't see "Stella E-Bikes" on the signage.

```suggestion
      "name": "Stella",
```